### PR TITLE
[AIRFLOW-4691](https://issues.apache.org/jira/browse/AIRFLOW-4691) Make tests/cli Pylint compatible

### PR DESCRIPTION
**Pull request by kik-kik**
### Description
- As part of [AIRFLOW-4364 Integrate Pylint](https://issues.apache.org/jira/browse/AIRFLOW-4364)

### Tests
- No code changes, only added docstring

### CODE REVIEW

**Comment by mik-laj** [comment]
Do the tests really require docstrings? It seems to me that it has been disabled.
https://github.com/apache/airflow/blob/10d39eff25657c78c2e4d99eb8f4a4779938b5c8/scripts/ci/in_container/_in_container_utils.sh#L178
https://github.com/apache/airflow/blob/47801057989046dfcf7b424ce54afee103803815/scripts/ci/in_container/run_pylint_tests.sh

File: tests/cli/test_cli.py [hunk 1]
```diff
@@ -79,6 +84,32 @@ def create_mock_args(  # pylint: disable=too-many-arguments
     raw=None,
     interactive=None,
 ):
+    """
+    creates mock arguments for a task
+    :param task_id:
+    :param dag_id:
+    :param subdir:
+    :param execution_date:
+    :param task_params:
```
Comment (kaxil) [inline tests/cli/test_cli.py:93]: This is as good as ignoring the test in the entire file !

Comment (kik-kik) [reply tests/cli/test_cli.py:93]: Sorry, do you mean I should remove them completely or add more information about each param?

Comment (kaxil) [reply tests/cli/test_cli.py:93]: Add more information about each parameter

**Pull request closed without merging**